tests/testBackground.py

```diff
@@ -513,22 +458,8 @@ def testOnlyOneGridCell(self):
                 testval = afwMath.cast_BackgroundMI(backobj).getPixel(bctrl.getInterpStyle(), xpix, ypix)
                 self.assertAlmostEqual(testval/mean, 1)
 
-    def testTicket1681OffByOne(self):
```

Comment:
@RobertLuptonTheGood disabled the code in fbbd59207be0c0181213e72ca6baa901f183d3a3 but I'm not sure why he left it around.